packages/genesys-spark-components/src/components/stable/gux-copy-to-clipboard/readme.md

```diff
@@ -14,23 +14,23 @@
-| `--gse-ui-copyToClipboard-label-text-lineHeight`               |                                                 |
-| `--gse-ui-copyToClipboard-label-text-textDecoration`           |                                                 |
-| `--gse-ui-copyToClipboard-tooltipIcon-error-foregroundColor`   |                                                 |
-| `--gse-ui-copyToClipboard-tooltipIcon-success-foregroundColor` |                                                 |
-| `--gse-ui-icon-size-sm`                                        |                                                 |
+| Name                                                                                                                   | Description |
+| ---------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `--gse-ui-copyToClipboard-contentContainer-gap`                                                                        |             |
+| `--gse-ui-copyToClipboard-iconContainer-padding`                                                                       |             |
+| `--gse-ui-copyToClipboard-label-active-backgroundColor`                                                                |             |
+| `--gse-ui-copyToClipboard-label-borderRadius`                                                                          |             |
+| `--gse-ui-copyToClipboard-label-foregroundColor`                                                                       |             |
+| `--gse-ui-copyToClipboard-label-padding`                                                                               |             |
+| `--gse-ui-copyToClipboard-label-text-fontFamily`                                                                       |             |
+| `--gse-ui-copyToClipboard-label-text-fontSize`                                                                         |             |
+| `--gse-ui-copyToClipboard-label-text-fontWeight`                                                                       |             |
+| `--gse-ui-copyToClipboard-label-text-lineHeight`                                                                       |             |
+| `--gse-ui-copyToClipboard-label-text-textDecoration`                                                                   |             |
+| `--gse-ui-copyToClipboard-tooltipIcon-error-foregroundColor`                                                           |             |
+| `--gse-ui-copyToClipboard-tooltipIcon-success-foregroundColor`                                                         |             |
+| `--gse-ui-icon-size-sm`                                                                                                |             |
+| `--gse-ui-tooltip-targetOffset - not using this token because the target offset is hardcoded to 16 within gux-tooltip` |             |
```

Comment:
There should be a ticket to remove this token (not part of v4 release)

Reply:
I will create the ticket, this reamdme must have just added when I ran the build as I did not change anything in that file.